mscore/editstyle.cpp

```diff
@@ -254,6 +254,10 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
       { Sid::harmonyFretDist,         false, harmonyFretDist,         0 },
       { Sid::minHarmonyDistance,      false, minHarmonyDistance,      0 },
       { Sid::maxHarmonyBarDistance,   false, maxHarmonyBarDistance,   0 },
+      { Sid::harmonyPlay,             false, harmonyPlay,             0 },
+      { Sid::harmonyVoiceLiteral,     false, voicingSelectWidget->interpretBox, 0 },
+      { Sid::harmonyVoicing,          false, voicingSelectWidget->voicingBox, 0 },
+      { Sid::harmonyDuration,         false, voicingSelectWidget->durationBox, 0 },
```

Comment:
No reset buttons?

Reply:
In Inspector at least voicingBox has a reset button
Oh, I see, that reset and set to style button applies to all 3 settings in one go